lighthouse-core/config/constants.js

```diff
@@ -83,6 +83,7 @@ const defaultPassConfig = {
   loadFailureMode: 'fatal',
   recordTrace: false,
   useThrottling: false,
+  pauseAfterFcpMs: 0,
```

Comment:
we should probably document sometime why these are defaulted to 0 in here but then all have non-0 defaults in `default-config.js` :)